tests/data/workspace-discovery/split_package/bad/pixi.toml

```diff
@@ -0,0 +1,12 @@
+[workspace]
+channels = [
+  "https://prefix.dev/pixi-build-backends",
+  "https://prefix.dev/conda-forge",
+]
+platforms = ["linux-64"]
+preview = ["pixi-build"]
+# name = "test_build_conda_package"
```

Comment:
Please also add a failing case with `channel`

Reply:
Channel is not derived though, its not part of this PR.